### DISCUSSION TITLE
Drumrolls with 0 duration are placeable

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#19808] [Q&A] [answered]
https://user-images.githubusercontent.com/53872073/185012021-76c848d8-d4f8-4d1d-82fe-9be7c14db4ba.mp4

Mapper is expected to drag to place a drumroll. But if you click and release mouse without dragging, drumroll still will be placed, with 0 duration.

**Comment by peppy** [comment, marked as answer]
Will be fixed via #19831.
